integration/case_simple.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"time"
+
+	"github.com/pingcap/log"
+	"github.com/pingcap/ticdc/integration/framework"
+	"go.uber.org/zap"
+)
+
+//nolint:unused
+type simple struct {
+	framework.AvroSingleTableTask
+}
+
+/*
+func newSimple() *simple {
+	simple := new(simple)
+	simple.AvroSingleTableTask.TableName = "test"
+	return simple
+}
+*/
+
+func (s *simple) Name() string {
+	return "simple"
+}
+
+func (s *simple) Run(ctx *framework.TaskContext) error {
+	_, err := ctx.Upstream.ExecContext(ctx.Ctx, "create table test (id int primary key, value int)")
+	if err != nil {
+		return err
+	}
+
+	for i := 0; i < 100; i++ {
+		_, err := ctx.Upstream.ExecContext(ctx.Ctx, "insert into test (id, value) values (?, ?)", i, i*10)
+		if err != nil {
+			return err
+		}
+	}
+
+	time.Sleep(10 * time.Second)
```

Comment:
how about use a retry mechanism instead of sleeping

Reply:
We have a retry mechanism encapsulated in the implementations of the Awaitable interface.